azure-pipelines.yml

```diff
@@ -102,5 +107,5 @@ jobs:
         - py38_test
         - py38_test_online
         - py38_test_32bit
-        - py38_test_oldestdeps
+        - py37_test_oldestdeps
```

Comment:
What's the reason for moving this to 3.7?

Reply:
My reasoning but cadair might have a different one. 

Oldest deps should be tested on the oldest python version we support.

Reply:
I didn't actually change the version, just needed to update the wheels depends list because someone forgot it.